src/create_disk.sh

```diff
@@ -209,7 +209,7 @@ if [ "${remote_name}" != NONE ]; then
 fi
 ostree pull-local "$ostree" "$ref" --repo rootfs/ostree/repo $remote_arg
 ostree admin os-init "$os_name" --sysroot rootfs
-allkargs='root=/dev/disk/by-label/root rootflags=defaults,prjquota rw $ignition_firstboot'
```

Comment:
Hmm, looks like we're losing `rw` here, was that on purpose?

Reply:
Yep, I don't think anything parses it without `root=` causing the systemd generator to run.  In any case it's cleaner to have our systemd unit do it.